### PR TITLE
Update ViewActions.cs

### PR DESCRIPTION
**Pull request by Nikitayfh**
Now you can zoom image without pressing CTRL key

### CODE REVIEW

File: Pinta.Core/Actions/ViewActions.cs [hunk 1]
```diff
@@ -140,11 +140,13 @@ public void CreateMainMenu (Gtk.Menu menu)
 			ImageMenuItem zoomin = ZoomIn.CreateAcceleratedMenuItem (Gdk.Key.plus, Gdk.ModifierType.ControlMask);
 			zoomin.AddAccelerator ("activate", PintaCore.Actions.AccelGroup, new AccelKey (Gdk.Key.equal, Gdk.ModifierType.ControlMask, AccelFlags.Visible));
 			zoomin.AddAccelerator ("activate", PintaCore.Actions.AccelGroup, new AccelKey (Gdk.Key.KP_Add, Gdk.ModifierType.ControlMask, AccelFlags.Visible));
+			zoomin.AddAccelerator ("activate", PintaCore.Actions.AccelGroup, new AccelKey (Gdk.Key.KP_Add, 0, AccelFlags.Visible));
```
Comment (cameronwhite) [inline Pinta.Core/Actions/ViewActions.cs:143]: Should there also be a matching accelerator for the `equal` key? (and `underscore` for the zoom out case). There are shortcuts with Ctrl for both.

**Review by cameronwhite — CHANGES_REQUESTED** [review]

File: Pinta.Core/Actions/ViewActions.cs (hunk 1 above)
Comment (Nikitayfh) [reply Pinta.Core/Actions/ViewActions.cs:143]: I tried to do the same with underscore key, but it doesn't work well without CTRL. Equal key works fine.

Comment (Nikitayfh) [reply Pinta.Core/Actions/ViewActions.cs:143]: Fixed it

**Review by cameronwhite — APPROVED** [review]
Looks good, thanks!

**Pull request merged**